src/main/resources/webapp/interface/types.d.ts

```diff
@@ -0,0 +1,44 @@
+import type {
+	Highlight,
+	QueryFilters
+} from '/lib/explorer/types.d';
+
+import {
+	GraphQLInterfaceSearchAggregation
+} from './types/Aggregation.d';
+
+
+export {
+	GraphQLInterfaceSearchAggregation,
+	AggregationType,
+	AggregationTypesObj
+} from './types/Aggregation.d';
+
+
+export interface CamelToFieldObj {
+	[key :string] :string
+}
+
+export interface SearchResolverEnv {
+	args :{
+		aggregations? :GraphQLInterfaceSearchAggregation
+		count :number
+		//after :string
+		filters? :QueryFilters
+		//first :number
+		highlight? :Highlight
+		searchString :string
+		start :number
+	}
+}
```

Comment:
In other places you assign `value: type` and not `value :type` why change the style from file to file?

Reply:
In Javascript I've always done name: value, so sometimes I forget doing name :type in Typescript. I fix that whenever I come across it.